packages/features/embed/Embed.tsx

```diff
@@ -299,7 +299,7 @@ const EmailEmbed = ({
         <Collapsible open>
           <CollapsibleContent>
             <div className="text-default mb-[9px] text-sm">{t("timezone")}</div>
-            <TimezoneSelect id="timezone" value={timezone} isDisabled />
+            <TimezoneSelect id="timezone" value={timezone} isDisabled={false} />
```

Comment:
@hariombalhara was this disabled on purpose?

Reply:
@Pradumn27 worked on this feature. I believe there has to be a reason. It isn't accidental.